src/responder/pac/pacsrv.c

```diff
@@ -147,12 +147,17 @@ int pac_process_init(TALLOC_CTX *mem_ctx,
                                    PAC_SBUS_SERVICE_NAME,
                                    PAC_SBUS_SERVICE_VERSION,
                                    MT_SVC_SERVICE,
-                                   &rctx->last_request_time, &rctx->mon_conn);
+                                   &rctx->last_request_time, &rctx->sbus_conn);
     if (ret != EOK) {
         DEBUG(SSSDBG_FATAL_FAILURE, "fatal error setting up message bus\n");
         goto fail;
     }
 
+    ret = sss_resp_register_sbus_iface(rctx->sbus_conn, rctx);
```

Comment:
Here and everywhere: previously this was done in 'responder_common.c'
Why not to keep it there, in `sss_process_init()`?

Reply:
sss_process_init connected to data providers and connection to monitor is not yet established. I think we can move `sss_monitor_service_init`  to sss_process_init to connect to the monitor and then register the interfaces there.